roles/aodh/templates/etc/serverspec/aodh_spec.rb

```diff
@@ -33,6 +28,18 @@
   it { should be_owned_by 'aodh' }
 end
 
+{% if inventory_hostname in groups['controller'] %}
+processes = ['aodh-api',
+  'aodh-evaluator',
+  'aodh-listener',
+  'aodh-notifier']
+processes.each do |process|
+  describe process(process) do
+    its(:user) { should eq 'aodh' }
```

Comment:
I don't think it's checking that the process is running, maybe there should be a better if block around that to not alert if it's not running, but more importantly it's checking that IF the process is running THAT it is running as the correct user.